include/nogdb_types.h

```diff
@@ -197,12 +200,15 @@ namespace nogdb {
 
     class Record {
     public:
+
+        using DataType = std::map<std::string, Bytes>;
```

Comment:
Is this a mapping between a property name and its associated data value in 'Bytes'? 'DataType' would be thought of as data type such as int, double, etc.